.github/workflows/build-cli-docker.yml

```diff
@@ -0,0 +1,166 @@
+---
+name: Build bws Docker image
+
+on:
+  push:
+    paths:
+      - "crates/bws/**"
+  workflow_call:
+    inputs:
+      sdk_branch:
+        type: string
+        default: master
+      is_workflow_call:
+        type: boolean
+        default: true
+  workflow_dispatch:
+    inputs:
+      sdk_branch:
+        description: "Server branch name to deploy (examples: 'master', 'rc', 'feature/sm')"
+        type: string
+        default: master
+  pull_request:
+    paths:
+      - ".github/workflows/build-cli-docker.yml"
+      - "crates/bws/**"
+
+env:
+  _AZ_REGISTRY: bitwardenprod.azurecr.io
+
+jobs:
+  build-docker:
+    name: Build Docker image
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout Repository - workflow_call
+        if: ${{ inputs.is_workflow_call == true }}
+        uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+        with:
+          repository: bitwarden/sdk
+          ref: master
```

Comment:
We don't need to check out different repos on different inputs. The second checkout step is sufficient 😃

```suggestion
```